lib/td_cache/domain_cache.ex

```diff
@@ -141,7 +142,16 @@ defmodule TdCache.DomainCache do
       [add_or_remove_root, @roots_key, id]
     ]
 
-    Redix.transaction_pipeline(commands)
+    {:ok, results} = Redix.transaction_pipeline(commands)
+
+    event = %{
+      event: "domain_updated",
+      domain: "domain:#{id}"
+    }
+
+    {:ok, _event_id} = Publisher.publish(event, "domain:events")
+
+    {:ok, results}
```

Comment:
Currently, the module `TdBg.Cache.DomainLoader` loads all domains into cache using `put_domain` when `td_bg` starts up. Events should only be emitted when the domain has changed (e.g. using the `updated_at` timestamp).